### PR TITLE
feat: support for linked wearables urns

### PR DESCRIPTION
**Pull request by marianogoldman**

### CODE REVIEW

File: src/resolvers.ts [hunk 1]
```diff
@@ -512,3 +530,111 @@ export async function resolveThirdPartyCollectionOnlyCollection(
...
+
+  if (contract) {
+    result = {
+      namespace: 'decentraland',
+      uri,
+      blockchain: 'ethereum',
+      type: 'blockchain-collection-linked-wearables-collection',
+      contractAddress: contract,
+      network: groups.network === 'ethereum' ? 'mainnet' : groups.network.toLowerCase(),
+      thirdPartyName: groups.linkedWearableProvider,
```
Comment (LautaroPetaccio) [inline src/resolvers.ts:575]: Should we standardise how we are calling to this property?
```suggestion
      linkWearableProvider: groups.linkedWearableProvider,
```

File: src/resolvers.ts [hunk 2]
```diff
@@ -512,3 +530,111 @@ export async function resolveThirdPartyCollectionOnlyCollection(
...
+      contractAddressChain: groups.contractAddressChain,
```
Comment (LautaroPetaccio) [inline src/resolvers.ts:576]: Should distinguish this property between the `contractAddress` one somehow?
```suggestion
      linkedWearableContractAddressChain: groups.contractAddressChain,
```

File: src/resolvers.ts [hunk 3]
```diff
@@ -454,8 +472,8 @@ export async function resolveThirdPartyCollection(
       blockchain: 'ethereum',
       type: 'blockchain-collection-third-party',
       network: groups.network === 'ethereum' ? 'mainnet' : groups.network.toLowerCase(),
-      thirdPartyName: groups.thirdPartyName,
-      collectionId: groups.collectionId,
+      linkedWearableProvider: groups.thirdPartyName,
+      LinkedWearableContractAddress: groups.collectionId,
```
Comment (aleortega) [inline src/resolvers.ts:476]: Should be camel case?

File: test/parse-urn.spec.ts [hunk 4]
```diff
@@ -11,78 +19,82 @@ function testValidUrnToInclude(urn: string, toInclude: Partial<DecentralandAsset
...
-      network: 'sepolia',
-      id: '0x1'
+    // it("test land (address)", async () => {
+    //   expect(await parseUrn("urn:decentraland:ethereum:0xf87e31492faf9a91b02ee0deaad50d51d56d5d4d:0x1")).toMatchObject({
+    //     contractAddress: "0xf87e31492faf9a91b02ee0deaad50d51d56d5d4d",
+    //     type: "blockchain-asset",
+    //     network: "mainnet",
+    //     id: "0x1",
+    //   })
+    // })
```
Comment (aleortega) [inline test/parse-urn.spec.ts:70]: Can we remove this test? it was commented out 3 years ago and it doesn't seem we are going to recover it.

**Pull request merged**